ethcore/src/state/mod.rs

```diff
@@ -121,12 +128,14 @@ impl AccountEntry {
 		AccountEntry {
 			account: self.account.as_ref().map(Account::clone_dirty),
 			state: self.state,
+			clean_balance: self.clean_balance,
```

Comment:
This is in line with the declaration order. Other constructors are exceptions because they use `account` before it is moved into `self.account`. Will fix here for consistency.